ppa/archive/models.py

```diff
@@ -11,10 +11,13 @@
 from django.contrib.auth.models import User
 from django.contrib.contenttypes.models import ContentType
 from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
 from django.db import models
 from django.urls import reverse
 from eulxml.xmlmap import load_xmlobject_from_file
 from flags import Flags
+from intspan import ParseError as IntSpanParseError
+from intspan import intspan
```

Comment:
wow, this is great! i feel like we encounter applications for this all the time; "page range" seems to come up often

Reply:
yeah, I was really pleased when I found it, makes our code much easier